pkg/install/install.go

```diff
@@ -237,3 +265,30 @@ func waitForCRDsEstablished() error {
 func waitForPodsReady(ns string) error {
 	return runCommand(exec.Command("kubectl", "wait", "pod", "--timeout=10m", "--for=condition=Ready", "-l", "!job-name", "-n", ns))
 }
+
+func runHelmInstall(registryAddress string) error {
+	cmd := exec.Command("helm", "install",
```

Comment:
Can we mark the exec command with linter ignore command just to avoid further golangci-linter errors.

Reply:
In addition I'd not mind checking if `helm` is installed first and displaying "Please install `helm` CLI" for a nicer UX.

Not a blocking issue though.